packages/next/src/cli/next-info.ts

```diff
@@ -31,7 +81,11 @@ async function getNextConfig() {
   }
 }
 
-function getBinaryVersion(binaryName: string) {
+/**
+ * Returns the version of the specified package manager.
+ * N/A if it's not installed.
+ */
+function getPackageManagerVersion(binaryName: 'npm' | 'yarn' | 'pnpm') {
```

Comment:
Why was this renamed from `getBinaryVersion`? I think it makes more sense as "binary" since its invoking the binary. Its not related to "package managers".

Reply:
ugh, looks like mistake while I was working on else and forgot to revert it back.